### DISCUSSION TITLE
docker-compose.yml vs. tpot.yml

### DISCUSSION THREAD

**Discussion by xcampax** [telekom-security/tpotce#986] [Q&A] [answered]
Hello, this is probably a stupid question but can someone please explain why according to tpot.yml docker images are pulled from dtagdevsec, but docker-compose.yml references ghcr.io/telekom-security? Thank you.

**Comment by t3chn0m4g3** [comment, marked as answer]
All images are built and served from Docker Hub. The ones in the individual compose files are remnants from when both DockerHub and GHCR were supported.

**Reply by xcampax** [reply]
Am I understanding correctly that the /opt/tpot/docker/docker-compose.yml file is not actively used anymore? I was wondering whether I needed to update the repo in this file...
Also, I am still seeing the ghcr.io images when executing 'docker images' - can I delete those?

**Reply by t3chn0m4g3** [reply]
`/opt/tpot/docker/docker-compose.yml` was only used to build locally, and will be updated to reflect DockerHub. Running T-Pot always uses `/opt/tpot/etc/tpot.yml`. Running `/opt/tpot/update.sh` will take you to the latest master.

**Reply by t3chn0m4g3** [reply]
Usage of GHCR was just suspended.

**Reply by xcampax** [reply]
Thank you!
